nprrelay/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine
+FROM alpine:3.20.2
```

Comment:
Similar came out of Pen Test review for Visits, should be locking Docker images to a specific version.